test/lua/unit/redis_tls.lua

```diff
@@ -0,0 +1,36 @@
+-- Simple unit test to validate TLS options propagation in lua_redis
```

Comment:
I'm sorry but we simply can't test Redis in unit tests, please remove it. We need some functional test but I can do it by myself.